### DISCUSSION TITLE
Unexpected error when trying to create one particular organization name that already exists

### DISCUSSION THREAD

**Discussion by dzek69** [npm/feedback#557] [Website] [answered]
When I try to create an organization, using a name that already exists I get:
> The organization name 'ns' is not available.
Which is perfectly fine and expected.

But one specific name - `nitra` - results in weird error:
> Creation denied. Please contact support at https://npmjs.com/support.

I assume there is some technical differences between those organizations leading to different "ending point" in the code, but I think both cases should get the same error message, like: `The organization name 'GIVEN_NAME' is not available.`

I contacted support and was told that I should report it here.
Another thing that is from the `moderator` point of view different errors happens as well:

Quote from support:
> I can confirm that the ns and nitra scopes are already in use, so your attempts to create an org with either name failed. When I tried to create an org with the nitra name, I received a "Creation failed" error, instead of an error stating that the scope was already in use.

**Comment by MylesBorins** [comment, marked as answer]
@dzek69 the reason you got the different errors is because `ns` is a User Account and `nitra` is an org

**Reply by dzek69** [reply]
Ok, it's nice to know a reason but I think it should be reworded anyway :)
